.github/workflows/stale.yml

```diff
@@ -12,10 +12,16 @@ jobs:
         id: stale-pr
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-pr-message: 'This PR is stale because it has been open 20 days with no activity. Remove stale label or comment or this will be closed in 10 days'
+          # Configure Issues
+          days-before-issue-stale: 180
+          days-before-issue-close: 30
```

Comment:
Shouldn't this also be 180? I don't want to automatically close them?

Reply:
It will go stale in 180 days which is 6 months, then once it goes stale you have another 30 days (or a month) to remove the stale label, or comment to keep it alive. If you do nothing, it will then close. So, basically an issue will close after 7 months of no activity.

Reply:
https://github.com/actions/stale#days-before-close

Reply:
🤯